Assets/Prefabs/WeaponsSystem/Projectiles/Enemy/EnemyKinematicProjectile.prefab

```diff
@@ -0,0 +1,146 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!1 &3998066587341369951
+GameObject:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  serializedVersion: 6
+  m_Component:
+  - component: {fileID: 3998066587341369946}
+  - component: {fileID: 3998066587341369945}
+  - component: {fileID: 3998066587341369944}
+  - component: {fileID: 3998066587341369947}
+  - component: {fileID: 3998066587341370276}
+  m_Layer: 0
+  m_Name: EnemyKinematicProjectile
+  m_TagString: EnemyProjectile
```

Comment:
Why do the enemies have their own Projectile prefab (which is the same as the other normal KinematicProjectile, just with a different tag)? 
And also, the fact that this projectile has a different tag ("EnemyProjectile" vs "Projectile") will cause some issues regarding interactables like the explosive barrel

Reply:
it's because otherwise they killed themselves

Reply:
iirc

Reply:
We could've modified the code itself, but thought it would be cleaner to not have a bunch of if statements in the code

Reply:
I don't think it warrants it's own prefab tho. There's a field in the ProjectileBehaviour called "shouldDamagePlayer" and you can use that during collision detection to know if the bullet was fired from an enemy.

Reply:
It's way more error prone to have it as its own prefab